src/applications/disability-benefits/all-claims/content/ancillaryFormsWizardSummary.jsx

```diff
@@ -89,6 +89,9 @@ const individualUnemployabilityContent = (
 );
 
 export default function summaryDescription({ formData }) {
+  const showUnemployableContent =
```

Comment:
I'm not sure this is the best way to get this to still show up/not show up depending on the outcome 🤔

Reply:
```
const showUnemployableContent =
    formData['view:unemployable'] &&
    formData['view:unemployabilityUploadChoice'] === 'answerQuestions';
```
Should it be like this for individualUnemployabilityContent to show up?

Reply:
Well the goal is to get it not show up if they run through our 8940/4192 form

Reply:
Oh! Got it!